adapters/invibes/invibestest/exemplary/no-ad.json

```diff
@@ -0,0 +1,64 @@
+{
+  "mockBidRequest": {
+    "id": "test-request-id",
+    "site": {
+      "page": "https://publisher.com/url"
+    },
+    "user": {
+      "buyerUid": "123"
+    },
+    "regs": {
+      "ext": {
+        "gdpr": 0
+      }
+    },
+    "imp": [
+      {
+        "id": "test-imp-id",
+        "banner": {
+          "format": [
+            {
+              "w": 300,
+              "h": 150
+            }
+          ]
+        },
+        "ext": {
+          "bidder": {
+            "placementId": "2eb6bd58-865c-47ce-af7f-a918108c3fd2"
+          }
+        }
+      }
+    ]
+  },
+  "httpCalls": [
+    {
+      "expectedRequest": {
+        "uri": "https://adweb.videostepstage.com/bid/VideoAdContent",
+        "body": {
+          "BidParamsJson": "{\"PlacementIds\":[\"2eb6bd58-865c-47ce-af7f-a918108c3fd2\"],\"BidVersion\":\"4\",\"Properties\":{\"2eb6bd58-865c-47ce-af7f-a918108c3fd2\":{\"Formats\":[{\"w\":300,\"h\":150}],\"ImpId\":\"test-imp-id\"}}}",
+          "Bvid": "",
+          "Gdpr": false,
+          "GdprConsent": "",
+          "Height": "",
+          "IsAmp": false,
+          "Kw": "",
+          "Lid": "123",
+          "Location": "https://publisher.com/url",
+          "IsTestBid": false,
+          "InvibBVLog": false,
+          "VideoAdDebug": false,
+          "Width": ""
+        }
+      },
+      "mockResponse": {
+        "status": 200,
+        "body": {
+          "VideoAdContentResult": {
+            "Ads": []
```

Comment:
The response object contains many properties which can change in time. We would prefer not to declare the properties here, because we don't want to modify the adapter every time the structure of the response object changes.

Reply:
VideoAdContentResult used in no-ad.json remained in this test file from a previos version of the adapter and make no sense in the current version (the response body doesn't contain this property anymore). I have removed it.